quiz/docker-compose.yaml

```diff
@@ -20,11 +20,52 @@ services:
     restart: on-failure
     networks:
       - proxy-web
+      - backend_network
     depends_on:
       - web
+      - backend_quiz
     ports:
       - "80:80"
+    volumes:
+      - ./backend/staticfiles:/app/staticfiles
```

Comment:
Nie do końca rozumiem po co są pliki statyczne wrzucane do proxy, proxy to proxy a nie web server dla staticów.
Wystaw je na endpoincie (a prawdopodobie django już je gdzieś wystawia) i ustaw go wewnąt proxy.